src/Main.java

```diff
@@ -1,5 +1,61 @@
+import java.time.LocalDate;
+
 public class Main {
+    public static void checkingLeapYear(int year2) {
+        if (year2 % 4 != 0) {
+            System.out.println(year2 + " год не является високосным.");
+        } else if (year2 % 100 == 0 && year2 % 400 != 0) {
+            System.out.println(year2 + " год не является високосным.");
+        } else {
+            System.out.println(year2 + " год является високосным.");
+        }
+        System.out.println();
+    }
+
+    public static void ResolvesInstallingOS(int clientOS2, int clientDeviceYear2) {
```

Comment:
Методы - в camelCase начиная с буквы в нижнем регистре
```suggestion
    public static void resolvesInstallingOS(int clientOS2, int clientDeviceYear2) {
```